src/main/java/org/gridsuite/network/map/model/TapChangerStepData.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
+
+/**
+ * @author Hugo Marcellin <hugo.marcelin at rte-france.com>
+ */
+@Builder
+@Getter
+@EqualsAndHashCode
+public class TapChangerStepData {
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Integer index;
```

Comment:
Why do you need an index here ?

Reply:
To enforce steps are displayed in the correct order in the front